.github/workflows/ci.yaml

```diff
@@ -131,3 +132,96 @@ jobs:
         with:
           name: kind-logs-${{ matrix.ip-family }}-${{ matrix.bgp-type }}-${{ matrix.deployment}}
           path: /tmp/kind_logs/
+
+  e2e-use-operator:
+    runs-on: ubuntu-20.04
+    defaults:
+      run:
+        shell: bash
+        working-directory: metallb
+    env:
+      metallb_operator_image: "metallb-operator:ci"
+    strategy:
+      fail-fast: false
+    steps:
```

Comment:
No need to rebuild.
I think you can save all the steps and just deploy using the single manifest: https://github.com/metallb/metallb-operator/blob/main/bin/metallb-operator-with-webhooks.yaml

Reply:
+1, right! but still need the sed part:
```
sed -i 's/quay.io\/metallb\/speaker:main/quay.io\/metallb\/speaker:dev-amd64/g' bin/metallb-operator-with-webhooks.yaml
sed -i 's/quay.io\/metallb\/controller:main/quay.io\/metallb\/controller:dev-amd64/g' bin/metallb-operator-with-webhooks.yaml
```